### PR TITLE
Update gitignore

### LINKED ISSUE #3: Add proper .gitignore

### PR DESCRIPTION
**Pull request by XPerianer**
Closes #3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,7 @@
 .idea/*
-*build*/
\ No newline at end of file
+*build*/
+libs/
```
Comment (NiklasKoehnecke) [inline .gitignore]: Isnt this a problem with the submodule?

Comment (XPerianer) [reply .gitignore]: Sorry, I don't get the point. Please clearify.

Comment (NiklasKoehnecke) [reply .gitignore]: libs/ has the gtest submodule in it. are you sure this is no problem? (Im not very experienced with submodule)

Comment (XPerianer) [reply .gitignore]: The right way to handle this seems to be that you can mark the submodules as 'dirty', that way any changes in the submodule won't be shown in the staging area. But this should actualy not be in the .gitignore so I removed it.
See also https://stackoverflow.com/questions/7912022/do-you-ignore-a-git-submodule-in-your-gitignore-or-commit-it-to-your-repo/7912101

**Pull request merged**